src/main/java/tech/allegro/wildsnake/product/model/Product.java

```diff
@@ -13,14 +13,16 @@
     private String name;
     private String imageUrl;
     private String description;
+    private long price;
```

Comment:
Nie jest to najlepsze rozwiązanie, ale do prostych zastosowań wystarcza. W tym wypadku cenę trzeba przechowywać w groszach i tylko podczas wyświetlania zmieniać na złotówki . Cena 1.99zł to to było by 199 / 100 podczas wyświetlania za pomoc jakiegoś js lub funkcji templatkowej.